app/Entities/Place/Location.php

```diff
@@ -36,4 +39,17 @@ public function getLatitude(): float
     {
         return $this->latitude;
     }
+
+    public static function fromString(?string $location): ?self
+    {
+        if (!empty($location) && preg_match('/^[0-9]+(\.[0-9]+)?,[0-9]+(\.[0-9]+)?$/', $location) == false) {
```

Comment:
const FROM_STRING_PATTERN = `'/^[0-9]+(\.[0-9]+)?,[0-9]+(\.[0-9]+)?$/`.

This method should return new self($longitude, $latitude); or `throw new \InvalidArgumentException('The location has an incorrect format');`